Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/SchemaRegistry.cs

```diff
@@ -0,0 +1,221 @@
+    /// </summary>
+    /// <example>/path/to/your/keystore.p12</example>   
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SslKeystoreLocation { get; set; }
+
+    /// <summary>
+    /// SSL keystore (PKCS#12) password.
+    /// </summary>
+    /// <example>foo</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SslKeystorePassword { get; set; }
+
+    /// <summary>
+    /// Schema JSON.
+    /// </summary>
+    /// <example>
+    /// {
+    ///   "fields": [
+    ///     {
+    ///       "name": "intField",
+    ///       "type": "int"
+    ///     },
+    ///     {
+    ///       "name": "longField",
+    ///       "type": "long"
+    ///     },
+    ///     {
+    ///       "name": "floatField",
+    ///       "type": "float"
+    ///     },
+    ///     {
+    ///       "name": "doubleField",
+    ///       "type": "double"
+    ///     },
+    ///     {
+    ///       "name": "booleanField",
+    ///       "type": "boolean"
+    ///     },
+    ///     {
+    ///       "name": "stringField",
+    ///       "type": "string"
+    ///     },
+    ///     {
+    ///       "name": "nullField",
+    ///       "type": "null"
+    ///     },
+    ///     {
+    ///       "name": "bytesField",
+    ///       "type": "bytes"
+    ///     },
+    ///     {
+    ///       "name": "enumField",
+    ///       "type": {
+    ///         "name": "Colors",
+    ///         "symbols": [
+    ///           "RED",
+    ///           "GREEN",
+    ///           "BLUE"
+    ///         ],
+    ///         "type": "enum"
+    ///       }
+    ///     },
+    ///     {
+    ///       "name": "arrayField",
+    ///       "type": {
+    ///         "items": "string",
+    ///         "type": "array"
+    ///       }
+    ///     },
+    ///     {
+    ///       "name": "mapField",
+    ///       "type": {
+    ///         "type": "map",
+    ///         "values": "int"
+    ///       }
+    ///     },
+    ///     {
+    ///       "name": "fixedField",
+    ///       "type": {
+    ///         "name": "FourBytes",
+    ///         "size": 4,
+    ///         "type": "fixed"
+    ///       }
+    ///     },
+    ///     {
+    ///       "name": "unionField",
+    ///       "type": [
+    ///         "null",
+    ///         "string"
+    ///       ]
+    ///     },
+    ///     {
+    ///       "name": "recordField",
+    ///       "type": {
+    ///         "fields": [
+    ///           {
+    ///             "name": "nestedField",
+    ///             "type": "string"
+    ///           }
+    ///         ],
+    ///         "name": "NestedRecord",
+    ///         "type": "record"
+    ///       }
+    ///     }
+    ///   ],
+    ///   "name": "sampleRecord",
+    ///   "namespace": "com.mycorp.mynamespace",
+    ///   "type": "record"
+    /// }
+    /// </example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SchemaJson { get; set; }
+
+
+    /// <summary>
+    /// Field values.
```

Comment:
Lets also mention that this should be in JSON array format then. And example is just one record here. Do we support an array of records?

Reply:
Maybe in the future there will be support for an array of records, but for now just one record in each task run. User can use Frends foreach task to produce multiple records in a single process run.